src/boards/include/boards/pimoroni_interstate75.h

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
```

Comment:
Should this be 2021 now? :man_shrugging:

Reply:
Probably, but that seems like a wider change than this specific PR, as most (if not all) of the SDK says 2020...

Reply:
I think the general convention is that the copyright year on a file reflects the year in which it was created? :man_shrugging:

Reply:
If that's the case, then whoops on behalf of all the board commits that we, Adafruit, and Sparkfun have pushed this year 😅 

In any case, let me know if you want it updating for these two board. The others would be best to do by someone as a separate PR I think.

Reply:
I'm gonna back out and let @kilograham decide - apologies for causing any confusion!